public/static/docs/command-reference/remote/index.md

```diff
@@ -95,17 +95,37 @@ url = /path/to/remote
 remote = myremote
 ```
 
-## Example: Add a default Amazon S3 remote and modify its region
+## Example: Add an Amazon S3 remote and modify its region
 
 > 💡 Before adding an S3 remote, be sure to
 > [Create a Bucket](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html).
 
 ```dvc
-$ dvc remote add -d mynewremote s3://mybucket/myproject
+$ dvc remote add mynewremote s3://mybucket/myproject
```

Comment:
@ivan thinks that

> no need to revert that -d for now

but I think given the other improvements to this doc it's best to remove it from here. Basically each example shows the most typical uses of all the subcommands 🙂

Reply:
Plus it seems @efiop is thinking about changing the behavior of `dvc remote add` so the first remote is set as `--default` by default, at which point we'll have to revisit these examples again anyway.

Reply:
Hmmm. Ruslan also likes having `-d` here "for consistency" (see https://github.com/iterative/dvc.org/pull/902#issuecomment-573534572). I'm not sure about this consistency argument, will have to review the other remote subcommand docs and check where `-d` is used in examples and where it isn't. But it's not clear to me what the purpose of having `-d` consistently everywhere is. Again, I think having variety in the examples is more valuable 🙂

Reply:
Probably the argument is that in 95% of the cases people setting up it for the first time and in 99% of those cases they need `-d`. So, we are optimizing the first time user experience with this.

Reply:
OK, it's no problem for me to make sure all (or most) of the `remote add` examples have `-d`.

But my counter-argument is that the goal of having multiple examples is precisely to not just address the use case of 95% (presumably) of users but to have a variety of interesting combinations (to show how flexible the tool is). The first example would usually be the simplest one, geared for the 99%, and they get more interesting from there.

Reply:
OK update on this! I reviewed all the 54  instances of `remote add` in the docs (some are just links) and added `-d` into some of them  (the ones with a context where it seems like this would be the first remote added to that example project).

**Note that this includes in the home page `/`:**

![image](https://user-images.githubusercontent.com/1477535/72484490-01b85300-37ca-11ea-81e5-6bda90945dc3.png)

Please note that most of the instances don't have `-d` because it wouldn't make sense (like in the list of remote types of the `dvc remote add/modify` cmd refs, et al.) In the future, if setting the first remote as default is the default behavior or `dvc add remote`, we should probably remove all of these `-d`, except if talking specifically about setting an extra remote as default.

Please see 4950d252.